tasks/preflight.yml

```diff
@@ -95,11 +95,19 @@
     - prometheus_version == "latest"
     - prometheus_binary_local_path | length == 0
 
+- name: "Get checksum list"
+  set_fact:
+    prometheus_checksums: "{{ lookup('url', 'https://github.com/prometheus/prometheus/releases/download/v' + prometheus_version + '/sha256sums.txt', wantlist=True) | list }}"
+  delegate_to: localhost
+  run_once: true
+  when:
+    - prometheus_binary_local_dir | length == 0
+
 - name: "Get checksum for {{ go_arch }} architecture"
   set_fact:
     prometheus_checksum: "{{ item.split(' ')[0] }}"
```

Comment:
It might be good to change this variable name to `__prometheus_checksum` or something similar to show that this is only internally used and limit confusion with `prometheus_checksums`

Reply:
Thinking about this, I think both of the checksum variables should be `__` to mark them internal.

Reply:
I agree.